### PR TITLE
Make sure it works for meas_date = None

### PR DESCRIPTION
**Pull request by jasmainak**
closes https://github.com/mne-tools/mne-bids/issues/22

### CODE REVIEW

File: tests/test_bids_meg.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                raw_fname=raw_fname, events_fname=events_fname,
+                output_path=output_path, event_id=event_id,
+                overwrite=True)
+
+    # let's do some modifications to meas_date
+    raw = mne.io.read_raw_fif(raw_fname)
+    raw.info['meas_date'] = None
+    data_path2 = _TempDir()
+    raw_fname2 = op.join(data_path2, 'sample_audvis_raw.fif')
+    raw.save(raw_fname2)
```
Comment (jasmainak) [inline tests/test_bids_meg.py]: @larsoner it seems that `raw.info['meas_date']` is somehow saved even if I set it to `None`. What should I do to remove all traces of it?

Comment (jasmainak) [reply tests/test_bids_meg.py]: actually never mind, some bug on my end

Comment (teonbrooks) [reply tests/test_bids_meg.py]: I think the `read_info` fcn falls back to `raw.info['meas_id']` if `raw.info['meas_date']` 
cf. https://github.com/mne-tools/mne-python/blob/master/mne/io/meas_info.py#L1220

**Comment by jasmainak** [comment]
hey @romquentin this PR is held up because of this issue on the MNE end: https://github.com/mne-tools/mne-python/issues/4949. If you want to help, feel free to make a PR to MNE to fix that issue :-) I might get to it in a few days if you don't beat me to it

**Comment by romquentin** [comment]
Sorry, I am not sure how to deal whit that... :)

**Comment by jasmainak** [comment]
let me merge this PR so that @teonbrooks can continue on the tests

**Pull request merged**